eng/config/PublishData.json

```diff
@@ -45,30 +45,30 @@
       "vsBranch": "rel/d16.3",
       "vsMajorVersion": 16
     },
-    "release/dev16.4-preview1-vs-deps": {
+    "release/dev16.4-preview2-vs-deps": {
       "nugetKind": ["Shipping", "NonShipping"],
       "version": "3.4.*",
       "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
       "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-      "channels": [ "dev16.4p1" ],
+      "channels": [ "dev16.4p2" ],
       "vsBranch": "rel/d16.4",
       "vsMajorVersion": 16
     },
-    "release/dev16.4-preview2-vs-deps": {
+    "release/dev16.4-vs-deps": {
```

Comment:
Do we need a -vs-deps branch?

Reply:
discussed offline - yes, @jasonmalinowski changes will require it.

Reply:
Actually it's not me after all because of my lightup work: we already have changes that are in master-vs-deps but not master. Maybe those can be backported but we'd have to figure out what those all are....which isn't happening right now. :-)